pkg/sqlite/deprecate.go

```diff
@@ -0,0 +1,66 @@
+package sqlite
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+
+	"github.com/operator-framework/operator-registry/pkg/registry"
+)
+
+type SQLDeprecator interface {
+	Deprecate() error
+}
+
+// BundleDeprecator removes bundles from the database
+type BundleDeprecator struct {
+	store   registry.Load
+	querier registry.Query
```

Comment:
Why use `querier` and `store` here separately? That seems potentially dangerous over time.

Reply:
can you elaborate on the danger?